desktop/main.js

```diff
@@ -14,6 +14,9 @@ const log = require('electron-log');
 const ELECTRON_EVENTS = require('./ELECTRON_EVENTS');
 const checkForUpdates = require('../src/libs/checkForUpdates');
 
+// This variable is injected into package.json by electron-builder via the extraMetadata field (specified in electron.config.js)
+const {isProduction} = Boolean(require('../package.json'));
```

Comment:
This is pretty confusing along with `isDev`. It looks like both `isDev` and `isProduction` could be true?

Reply:
No, they can't both be true. I agree it's confusing though.

Reply:
Going to resolve this just to consolidate threads.